README.md

```diff
@@ -11,7 +11,7 @@ Requirements
 ------------
 
 -	[Terraform](https://www.terraform.io/downloads.html) 0.10.x
--	[Go](https://golang.org/doc/install) 1.8 (to build the provider plugin)
+-	[Go](https://golang.org/doc/install) 1.13 (to build the provider plugin)
```

Comment:
I wouldn't mind getting rid of this line and keeping everything above the fold targeted toward users (not developers). We specify the 1.13 requirement down below.

Reply:
Yeah, I can understand this change. I kind of pondered that a bit earlier as well.